Attribution Reporting.md

```diff
@@ -0,0 +1,187 @@
+# Attribution Reporting
+## Overview
+Attribution reporting is a strategy that allows marketers and sales teams to track the impact of their marketing efforts on specific goals, such as purchases or sales.  Traditionally, ad conversion measurement relied on third-party cookies, which could be used to track users across sites and raised privacy concerns. To facilitate these measurements without tracking, a new reporting API is required.  Microsoft Edge is supporting a limited version of reporting, with reports being sent per-event. Future work includes replacing these event-level reports with aggregation reporting.
+
+# Attribution Sources
+There are two types of attribution sources - navigation and event sources.  An attribution source is eligible for reporting if any page on any of the associated destination sites (advertiser sites) triggers attribution for the associated reporting origin.
+
+## Navigation Sources (Click)
```

Comment:
Are these examples illustrative, or is the "attributionsrc with URL" style the sole supported option?